ubcpi/static/html/ubcpi_edit.html

```diff
@@ -0,0 +1,153 @@
+                    <span class="tip setting-help">{{ esc.algos[esc.data.algo] }}</span>
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-algo-num-responses">Number of Responses<span class="tip setting-help">The number of responses presented to the students. Enter # for the same number as options.</span></label>
+                        <input class="input setting-input pi-options" name="pi-algo-num-responses" id="pi-algo-num-responses" ng-model="esc.data.algo.num_responses" ng-model-options="{ debounce: 500 }" type="text" required />
+                    </div>
+                </li>
+
+                <li class="field comp-setting-entry is-set">
+
+                    <label class="label setting-label" for="">Seeded Answers<span class="tip setting-help">Provide answers which can be shown to the students answering this question when there are not enough submitted responses from which to choose. </span><input type="button" class="button action-primary add-image-button" value="Add Seed" ng-click="esc.addSeed()" name="pi-option-add-seed" /></label>
+
+
+                    <div class="ubcpi-options-list-container">
+                        <div class="wrapper-comp-setting ubcpi-options-list seeded-answer" id="seed-{{ $index + 1 }}" ng-repeat="option in esc.data.seeds track by $index">
+                            <label class="label setting-label pi-setting-label" for="pi-option-seeded-{{$index}}">Seeded Answer {{$index + 1}} <span class="tip setting-help">Choose which option this seeded answer is for</span></label>
+                            <select name="pi-option-seeded-{{$index}}" id="pi-option-seeded-{{$index}}" ng-model="esc.data.seeds[$index].answer" ng-options="esc.data.options.indexOf(option) as ('Option ' + (esc.data.options.indexOf(option) +1) ) for option in esc.data.options" required></select>
+                            <a href="" class="remove-action remove-setting" aria-label="Remove seed" data-lang="af" data-value="" ng-click="esc.deleteSeed($index)" data-tooltip="Remove seed"><i class="icon fa fa-times-circle"></i><span class="sr">Remove</span></a>
+                            <label class="label setting-label pi-setting-label" for="pi-seed-{{$index}}">Rationale for Seeded Answer {{$index + 1}}<span class="tip setting-help">Provide the rationale for this seeded answer</span></label>
+                            <textarea class="input setting-input ubcpi-field" name="" id="pi-seed-{{$index}}" ng-model="esc.data.seeds[$index].rationale" ng-model-options="{ debounce: 500 }" required></textarea>
+                        </div>
+                    </div>
+
+                    <div ng-messages='piForm.$errors' ng-messages-multiple>
+                        <div class="seed_error" ng-message="seed_error">
+                            <p role="alert"><i class="icon fa fa-remove"></i> {{piForm.$errors.seed_error}}</p>
```

Comment:
The default state of the block has an error. Can you make it so when you add a new PI block, it passes validation by default?

![screen shot 2015-09-16 at 2 27 25 pm](https://cloud.githubusercontent.com/assets/1985317/9914300/28e5695c-5c7f-11e5-9cee-d16782179640.png)